### PR TITLE
Clarify Worker activity limit options

### PR DESCRIPTION
**Pull request by lorensr**
TODO: edit further if `TaskQueueActivitiesPerSecond` is per shard. cc @robholland

### CODE REVIEW

**Review by flossypurse — APPROVED** [review]
👍

File: docs/operation/how-to-tune-workers.md [hunk 1]
```diff
@@ -87,10 +87,10 @@ then consider increasing the number of pollers by adjusting `maxConcurrentWorkfl
 
 ### Rate Limiting
 
-If, after adjusting the poller and executors count as specified above, you still observe an elevated `schedule_to_start`, underutilized Worker hosts, or high `worker_task_slots_available`, you may want to check
+If, after adjusting the poller and executors count as specified above, you still observe an elevated `schedule_to_start`, underutilized Worker hosts, or high `worker_task_slots_available`, you may want to check:
```
Comment (djmagee) [inline docs/operation/how-to-tune-workers.md:90]:
```suggestion
If, after adjusting the poller and executors count as specified earlier, you still observe an elevated `schedule_to_start`, underutilized Worker hosts, or high `worker_task_slots_available`, you might want to check the following:
```

File: docs/operation/how-to-tune-workers.md [hunk 2]
```diff
@@ -87,10 +87,10 @@ then consider increasing the number of pollers by adjusting `maxConcurrentWorkfl
...
 
-1. If server-side rate limiting per Task Queue is set by `WorkerOptions#maxTaskQueueActivitiesPerSecond` and remove the limit or adjust the value up.
-2. If Worker-side rate limiting per Worker is set by `WorkerOptions#maxWorkerActivitiesPerSecond` and remove the limit. [GoSDK only]
+1. If server-side rate limiting per Task Queue is set by `WorkerOptions#maxTaskQueueActivitiesPerSecond`, remove the limit or adjust the value up. (See [Go](/docs/go/how-to-set-workeroptions-in-go/#taskqueueactivitiespersecond) and [Java](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/worker/WorkerOptions.Builder.html).)
```
Comment (djmagee) [inline docs/operation/how-to-tune-workers.md:92]: Should this list be unordered instead of ordered? It doesn't appear to be a sequence.

Comment (lorensr) [reply docs/operation/how-to-tune-workers.md]: I think unordered. Edited.

**Comment by lorensr** [comment]
It's not per shard, it's global. (Value is divided by the number of shards and each shard has a value/nShards limit)

https://github.com/temporalio/temporal/blob/aa17a4aba898a43d2bd74c94088b69348bc275d1/service/matching/matcher.go#L295

**Pull request merged**